openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1942,7 +1942,6 @@ def _assert_existing_user_error(self, response):
         for conflict_attribute in ["username", "email"]:
             self.assertIn(conflict_attribute, errors)
             self.assertIn("belongs to an existing account", errors[conflict_attribute][0]["user_message"])
-        self.assertNotIn("partial_pipeline", self.client.session)
```

Comment:
@UmanShahzad Again, with this change the test doesn't seem identical to what it was - are there a replacement?